### DISCUSSION TITLE
FN1 MyOpen Math

### DISCUSSION THREAD

**Discussion by tdegeorge** [TeamBasedInquiryLearning/precalculus#344] [General]
Reviewer: @AbbyANoble 

UPDATED on MOM

Outline of Questions:
Questions 1 - 4: Find the domain and range of a relation (mixed representations: ordered pairs, table, graphs)
Questions 5 - 13: Determine whether relations are functions for ordered pairs, tables, and some graphs/maps (note that these do not mention vertical line tests)
Questions 14 - 21: Using vertical line test to determine if a function. 
Question 22: Determine if the given equation is a function. Could not find many of these that weren't static...
Questions 23 - 32: Determine the domain from the equation. The first few start by asking what values are excluded from the domain and then the rest ask for the domain in interval notation.

**Comment by siwelwerd** [comment]
>I saw that there were some "problems" that were just media (i.e., videos).

How accessible are these videos?

**Reply by tdegeorge** [reply]
They look like YouTube videos embedded in the questions....so, closed caption but not sure what else...

**Comment by AbbyANoble** [comment]
Like instead of a question it was just a video to watch? And nothing to do? I'd vote no on including those. Or at least I'd probably take them out for my personal classes. That's something I'd post in D2L if it was useful, not in the homework. 

But I did include many questions that had videos as a resource to help with the question.

**Reply by tdegeorge** [reply]
Yeah, there are some that are just videos. I don't think I used those....I did find a lot with a video attached

**Comment by AbbyANoble** [comment]
There are a quite a few static ones here. Not sure if we want to include those!

**Reply by tdegeorge** [reply]
Removed all the static ones!

**Comment by AbbyANoble** [comment]
I'd move 21 up to the beginning of the vertical line test set of questions since the video is in the question instead of a link at the bottom.

**Comment by AbbyANoble** [comment]
I also agree that we probably don't need quite that many vertical line test questions. Especially since some of them have multiple questions within the question.

**Reply by AbbyANoble** [reply]
26 is static, so it could go. Also, 17 isn't technically static, but the graph just moves a little bit and nothing else really changes.

**Comment by AbbyANoble** [comment]
27 and 28 --  I think it's okay that there aren't more of these since each one is really multiple questions.
